testnet-contracts/scripts/sendLockTx.js

```diff
@@ -0,0 +1,126 @@
+module.exports = async () => {
+  /*******************************************
+   *** Set up
+   ******************************************/
+  let Web3 = require("web3");
+  var HDWalletProvider = require("@truffle/hdwallet-provider");
+
+  // Contract abstraction
+  const truffleContract = require("truffle-contract");
+  let contract = truffleContract(require("../build/contracts/Peggy.json"));
+
+  /*******************************************
+   *** Constants
+   ******************************************/
+  // Lock transaction default params
+  const DEFAULT_COSMOS_RECIPIENT =
+    "0x636f736d6f7331706a74677530766175326d35326e72796b64707a74727438383761796b756530687137646668";
+  const DEFAULT_COIN_DENOM = "0x0000000000000000000000000000000000000000";
+  const DEFAULT_AMOUNT = 10;
+
+  // Config values
+  const NETWORK_ROPSTEN =
+    process.argv[4] === "--network" && process.argv[5] === "ropsten";
+  const DEFAULT_PARAMS =
+    process.argv[4] === "--default" ||
+    (NETWORK_ROPSTEN && process.argv[6] === "--default");
+  const NUM_ARGS = process.argv.length - 4;
+
+  /*******************************************
+   *** Command line argument error checking
+   ***
+   *** truffle exec lacks support for dynamic command line arguments:
+   *** https://github.com/trufflesuite/truffle/issues/889#issuecomment-522581580
+   ******************************************/
+  if (NETWORK_ROPSTEN && DEFAULT_PARAMS) {
```

Comment:
a `switch` statement here would improve readability for the outer conditions

Reply:
Planning on refactoring to node scripts instead of using truffle exec due to the difficulties parsing/validating command line arguments and also its latency (script runs for ~25 seconds despite logging the LogLock information right away). This section will be deprecated shortly in favor of simpler input validation in the new script.